js/repl/src/Repl.js

```diff
@@ -160,6 +162,27 @@ export default class Repl extends React.Component {
     );
   }
 
+  _setupBabel() {
+    loadBabel(this.state.babel, success => {
```

Comment:
We _could_ refactor this alongside loadPlugins to be a bit more generic, but figured we could do that in a follow up, if anything.